plinth/modules/tor/__init__.py

```diff
@@ -98,7 +98,7 @@ def _diagnose_control_port():
 def _diagnose_url_via_tor(url, kind=None):
     """Diagnose whether a URL is reachable via Tor."""
     result = action_utils.diagnose_url(url, kind=kind, wrapper='torsocks')
-    result[0] = _('Access URL {url} on tcp{kind} via Tor') \
+    result[0] = _('Access URL {url} on tcp {kind} via Tor') \
```

Comment:
The outcome of the string should be 'tcp4' or 'tcp6'.  So, the original is correct.